public/views/index.html

```diff
@@ -2,7 +2,7 @@
 		<!-- Sidebar -->
 		<div id="sidebar-container" class="sidebar">
 			<div id="logout" class="sidebar" ng-model="user">
-				<a href="/">logout</a> | {{user.email}}
+				<a href="/" ng-click="logout()">logout</a> | {{user.email}}
```

Comment:
is it actually smarter to redirect to logout? I'm actually not sure. We have two options:

1. Redirect to /logout and have that GET request log the user out and then redirect to login.
2. Send an AJAX POST request to log the user out, then client side redirect to login.

Right now we're doing option 2, which I think is correct, but I'm not sure.